src/utilities.ts

```diff
@@ -14,7 +14,14 @@ export function getHops(request: Request, maxHops: number) {
 }
 
 export function sortOrderbookBySerializedPrice(orderbook: Orderbook) {
-  const json = orderbook.toJSON()
+  // TODO: previously one offer looked liked this in json
+  // {"price":9090936436582474,"volume":0.002000049787682}
+  // but now it looks like this
+  // {"volume":{"numerator":"e8d620e991","denominator":"01c6bf52634000"},
+  // "price":{"numerator":"05679e9e863369d800","denominator":"2ad7"}}
+  // Is it important that it doesn't change format? The documentation doesn't specify
+  // the format.
+  const json = orderbook.getOffers()
   json.bids.sort(sortDescending)
   json.asks.sort(sortAscending)
   return json
```

Comment:
TODO

Reply:
Yes, the frontend relies on having volume and price as a number and not an object type. The e2e test should already be testing that. Do they not?